imxvpuapi/imxvpuapi_vpulib.c

```diff
@@ -665,9 +667,9 @@ void imx_vpu_calc_framebuffer_sizes(ImxVpuColorFormat color_format, unsigned int
 	 * while in the chroma_interleave == 1 case, there is one shared chroma plane
 	 * for both Cb and Cr data, with cbcr_size bytes */
 	calculated_sizes->total_size = calculated_sizes->y_size
-	                             + (chroma_interleave ? calculated_sizes->cbcr_size : (calculated_sizes->cbcr_size * 2))
-	                             + calculated_sizes->mvcol_size
-	                             + alignment;
+								 + (chroma_interleave ? calculated_sizes->cbcr_size : (calculated_sizes->cbcr_size * 2))
+								 + calculated_sizes->mvcol_size
+								 + alignment;
 
```

Comment:
Again, alignment -> spaces. (I am aware that sometimes I did use tabs for alignment in cases like these. That was an inconsistency of mine. Stick to the alignment=spaces indentation=tabs rule.)